third_party/terraform/website/docs/guides/provider_reference.html.markdown

```diff
@@ -91,6 +104,9 @@ region is specified on a regional resource, it will take precedence.
 zone should be within the default region you specified. If another zone is
 specified on a zonal resource, it will take precedence.
 
+* `impersonate_service_account` - (Optional) The service account to impersonate for all Google API Calls.
```

Comment:
Do we need to add an `impersonate_service_account_delegates` as well?

Reply:
Delegates is a complex configuration and in most cases direct impersonation is generally sufficient. It is documented at the bottom